api/src/pcapi/routes/native/v1/subscription.py

```diff
@@ -43,7 +43,7 @@ def next_subscription_step(
     return serializers.NextSubscriptionStepResponse(
         next_subscription_step=user_subscription_state.next_step,
         stepper_includes_phone_validation=subscription_api.is_phone_validation_in_stepper(user),
-        allowed_identity_check_methods=subscription_api.get_allowed_identity_check_methods(user),
+        allowed_identity_check_methods=subscription_api.get_allowed_identity_check_methods(user),  # Deprecated
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏Mettre un commentaire plus précis. 
Par exemple: 
```python
# FIXME: (name, date) deprecated. Remove this field when :
# [ ] it is not used anymore in the frontend
# [ ] a forced updated happened
```

Eventuellement un lien vers le ticket front pour clean